specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -0,0 +1,4002 @@
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/workloadNetworks/default/portMirroringProfiles/{portMirroringId}": {
+      "get": {
+        "operationId": "WorkloadNetworks_GetPortMirroring",
+        "tags": [
+          "WorkloadNetworks"
+        ],
+        "summary": "Get a port mirroring profile by id in a private cloud workload network.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/portMirroringId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/WorkloadNetworkPortMirroring"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkloadNetworks_GetPortMirroring": {
+            "$ref": "./examples/WorkloadNetworks_GetPortMirroringProfiles.json"
+          }
+        }
+      },
+      "put": {
+        "operationId": "WorkloadNetworks_CreatePortMirroring",
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "WorkloadNetworks"
+        ],
+        "summary": "Create a port mirroring profile by id in a private cloud workload network.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/portMirroringId"
+          },
+          {
+            "name": "workloadNetworkPortMirroring",
+            "description": "NSX port mirroring",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/WorkloadNetworkPortMirroring"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
201 for create, 200 for update.  Please make a pass through and ensure this is correct + the async status codes are correct for all operations (commented a few times on them)

Reply:
Fixed 👍
Updated all workloadNetworks/xyz PUTs to include 201 status codes.